gtk4/src/drop_target.rs

```diff
@@ -2,11 +2,35 @@
 
 use crate::DropTarget;
 use glib::signal::connect_raw;
+use glib::Type;
 use glib::{translate::*, ObjectType, SignalHandlerId};
 use std::boxed::Box as Box_;
 use std::mem::transmute;
 
 impl DropTarget {
+    #[doc(alias = "gtk_drop_target_set_gtypes")]
+    pub fn set_gtypes(&self, types: &[Type]) {
+        let types: Vec<glib::ffi::GType> = types.iter().map(|t| t.to_glib()).collect();
+        unsafe {
+            ffi::gtk_drop_target_set_gtypes(
+                self.to_glib_none().0,
+                mut_override(types.as_ptr()),
```

Comment:
Can't do `to_glib_none()` here?

Reply:
Sadly not, we don't have a default impl  for `ToGlibContainerFromSlice` for usize.

Reply:
Oh well!